frank-flow/src/frontend/src/app/shared/services/node-generator.service.ts

```diff
@@ -143,13 +149,24 @@ export class NodeGeneratorService {
             }
           }
         }
+      } else {
+        this.forwards.push(
+          new Forward(
+            pipe.uid,
+            index + 1 === pipes.length
+              ? exits.length > 0
+                ? exits[0].uid
+                : 'implicitExit'
+              : pipes[index + 1]?.uid
```

Comment:
Would this work?

```suggestion
            index + 1 === pipes.length
              ? exits[0].uid ?? 'implicitExit'
              : pipes[index + 1]?.uid
```

Would be a bit more readable.